JacksonDunstanNativeCollections/NativePerJobThreadIntPtrs.cs

```diff
@@ -0,0 +1,434 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="NativePerJobThreadIntPtrs.cs" company="Jackson Dunstan">
+//     Copyright (c) Jackson Dunstan. See LICENSE.txt.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Unity.Burst;
+using Unity.Collections;
+using Unity.Collections.LowLevel.Unsafe;
+using Unity.Jobs.LowLevel.Unsafe;
+
+namespace JacksonDunstan.NativeCollections
+{
+	/// <summary>
+	/// Same as <see cref="NativeIntPtr"/>, but uses an array of integers
```

Comment:
It'd be better to describe the type in absolute terms so that readers don't need to also be familiar with another type.